File: DailyFlash Solutions/DailyFlash Solutions/19-DailyFlash_Solutions/4_Feb_Solutions_Three/Java/prog5.java
import java.util.Scanner;
class Demo {
	public static void main(String[] args) {
		int sum = 0;
		System.out.println("Enter num : ");
		Scanner sc = new Scanner(System.in);
		int num = sc.nextInt();
		for(int i = 1; i <= Math.sqrt(num); i++) {
			if(num % i == 0) {
				if(num / i == i) {
					System.out.print(i + " ");
					sum += i;
				} else {
					System.out.print(i + " " + num / i + " "); 
					sum += i;
					sum += num / i;
				}
			}
		}

		System.out.println("\nsum : " + sum);
	}
}

File: DailyFlash Solutions/DailyFlash Solutions/19-DailyFlash_Solutions/4_Feb_Solutions_Three/C/prog5.c
#include<stdio.h>
#include<math.h>
int main(void) {
	int num, sum = 0;
	printf("Enter num : ");
	scanf("%d", &num);
	for(int i = 1; i <= sqrt(num); i++) {
		if(num % i == 0) {
			if(num / i == i) {
				printf("%d ", i);
				sum += i;
			} else {
				printf("%d %d ", i, num / i);
				sum += i;
				sum += num / i;
			}
		}
	}

	printf("\nsum = %d", sum);
	return 0;
}
